### PR TITLE
[chore] Change comment on `bucket_counts` to reflect spec.

### PR DESCRIPTION
**Pull request by kovrus**
The comment on `bucket_counts` contains an invalid field name and the bucket boundaries are documented to be `[base^(offset+i), base^(offset + i + 1))` instead of `(base^(offset+i), base^(offset + i + 1)]`, see https://opentelemetry.io/docs/reference/specification/metrics/data-model/#exponential-buckets

### CODE REVIEW

**Comment by kovrus** [comment]
@specs-metrics-approvers Can you please review the PR?

**Review by reyang — APPROVED** [review]

File: opentelemetry/proto/metrics/v1/metrics.proto
Comment (reyang) [inline opentelemetry/proto/metrics/v1/metrics.proto:541]: Not blocking this PR, do we feel this might still be confusing since there are positive AND **negative** buckets?

**Review by jsuereth — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

File: opentelemetry/proto/metrics/v1/metrics.proto
Comment (bogdandrutu) [inline opentelemetry/proto/metrics/v1/metrics.proto:540]:
```suggestion
    // bucket_counts is an array of count values, where bucket_counts[i] carries
    // the count of the bucket at index (offset+i). bucket_counts[i] is the count
```

**Review by bogdandrutu — APPROVED** [review]

File: opentelemetry/proto/metrics/v1/metrics.proto
Comment (bogdandrutu) [reply opentelemetry/proto/metrics/v1/metrics.proto:540]: @open-telemetry/technical-committee since I cannot write suggestion and merge by myself (I can, but would be strange) would any of you look at this, apply suggestion and merge this PR?

Comment (tigrannajaryan) [reply opentelemetry/proto/metrics/v1/metrics.proto]: Done. I never used that button, please check the end result is what we need.

**Review by tigrannajaryan — APPROVED** [review]

File: opentelemetry/proto/metrics/v1/metrics.proto
Comment (tigrannajaryan) [reply opentelemetry/proto/metrics/v1/metrics.proto:541]: I am going to merge the PR, please open an issue if you feel it is necessary.

**Pull request merged**
